### ISSUE TITLE
odt->pdf converter throws java.io.UnsupportedEncodingException: Identity-H with Symbol font

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#273]
```
The odt->pdf converter throws java.io.UnsupportedEncodingException: Identity-H 
when a Symbol font is used in the odt.
```

Original issue reported on code.google.com by `angelo.z...@gmail.com` on 4 Jun 2013 at 12:49

**Comment by GoogleCodeExporter** [comment]
```
See the commit 
https://code.google.com/p/xdocreport/source/detail?r=038b282dcd887306a50fa0a40f1
7e0a46b15c7ec 

I have created a simple JUnit Issue273 which converts odt->pdf which contains a 
Symbol font.
```

Original comment by `angelo.z...@gmail.com` on 4 Jun 2013 at 12:56

**Comment by GoogleCodeExporter** [comment]
```
fixed
```

Original comment by `abe...@gmail.com` on 6 Jun 2013 at 12:40
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I get similar exception - what was fix for this? Thanks!
```

Original comment by `klemenzi...@gmail.com` on 19 May 2014 at 8:25

**Comment by GoogleCodeExporter** [comment]
```
Which version do you use? This issue was fixed on 1.0.4 if I remember.
```

Original comment by `angelo.z...@gmail.com` on 19 May 2014 at 8:34
